src/main/cpp/t06_longest.cpp

```diff
@@ -17,10 +17,22 @@
 
 #include "t06_longest.h"
 #include <iostream>
-
+#include <locale>
+#include <cstring>
 
 using namespace std;
 
 int t06_longest() {
-
+string s;
+int beq=0,max,lmax=-1,end,l;
+while (getline(cin,s)){
+    while (end = s.find(" ",beq) != string::npos){
+        l=end-beq;
+    if (l>lmax) {max=beq;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'max' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4719848)